### DISCUSSION TITLE
Help: Request error: Run failed: [openai] Rate Limit Error, Error code: 429 - {'error': {'message': 'Rate limit reached for gpt-4o-mini in organization org-FfR8gNQubFpGlLWIGooedD9O on requests per day (RPD): Limit 10000, Used 10000, Requested 1. Please try again in 8.64s. Visit https://platform.openai.com/account/rate-limits to learn more.', 'type': 'requests', 'param': None, 'code': 'rate_limit_exceeded'}}

### DISCUSSION THREAD

**Discussion by AIIA-Creator** [langgenius/dify#11806] [Help]
### Self Checks

- [X] I have searched for existing issues [search for existing issues](https://github.com/langgenius/dify/issues), including closed ones.
- [X] I confirm that I am using English to submit this report (我已阅读并同意 [Language Policy](https://github.com/langgenius/dify/issues/1542)).
- [X] [FOR CHINESE USERS] 请务必使用英文提交 Issue，否则会被关闭。谢谢！:）
- [X] Please do not modify this template :) and fill in all the required fields.

### 1. Is this request related to a challenge you're experiencing? Tell me about your story.

Run failed: [openai] Rate Limit Error, Error code: 429 - {'error': {'message': 'Rate limit reached for gpt-4o-mini in organization org-FfR8gNQubFpGlLWIGooedD9O on requests per day (RPD): Limit 10000, Used 10000, Requested 1. Please try again in 8.64s. Visit https://platform.openai.com/account/rate-limits to learn more.', 'type': 'requests', 'param': None, 'code': 'rate_limit_exceeded'}}

But still have the available
![image](https://github.com/user-attachments/assets/6477b924-1227-4fb7-91fe-fe3493087b81)

**Comment by rafael-keller** [comment]
Same here, i think the QUOTA have an APIKEY on DIFY platafrom (Cloud) and this key exceed limit rate, so chenge to you APIKEY and everything works fine...
